src/components/RecentSubmission.js

```diff
@@ -3,16 +3,20 @@ import PropTypes from 'prop-types';
 import './RecentSubmission.css';
 
 const RecentSubmission = (props) => {
-  return (
-    <div className="RecentSubmission">
-      <h3>The Most Recent Submission</h3>
-      <p className="RecentSubmission__submission">{  }</p>
+  if (!props.isSubmitted && props.submission) {
+    return ( 
+    <div className = "RecentSubmission" >
+      <h3> The Most Recent Submission </h3> 
+      <p className = "RecentSubmission__submission" > {props.submission} </p> 
     </div>
-  );
-}
+    )
+  } else {
+    return (null)
+  }}
 
 RecentSubmission.propTypes = {
   submission: PropTypes.string.isRequired,
+  isSubmitted: PropTypes.bool.isRequired,
```

Comment:
Also if you're not using `submission` as a prop, you should change it.  You're getting a warning because this is required, but you're not using it.